data/brands/shop/mobile_phone.json

```diff
@@ -1043,6 +1043,17 @@
         "shop": "mobile_phone"
       }
     },
+    {
+      "displayName": "Total by Verizon",
+      "locationSet": {"include": ["us"]},
+      "matchNames": ["Total Wireless"],
+      "tags": {
+        "brand": "Total by Verizon",
+        "brand:wikidata": "Q9089355",
```

Comment:
https://www.wikidata.org/wiki/Q64590606 seems better

Reply:
I don't know much about this, which should be used? Total is still a brand of TracFone. https://www.tracfonewirelessinc.com/en/brands/